### DISCUSSION TITLE
EQ point Hz based! (Why people is not intertacting this amazing plugin??? HELL)

### DISCUSSION THREAD

**Discussion by udeveloping** [ZL-Audio/ZLEqualizer#130] [Ideas]
Hello guys, i hope you can read my post, i have tried this amazing plugin! I would like to know you will add the point frequency to each eq point! Right there is just a Note based point like, if you wanna eq bass you need to go to B1 to Eq, but i think freq number could help to achieve a more precise eq! 

By the way which image format use the gui? o would like to propose a GUI for knobs and buttons!

Peace and keepo th f... good job!

**Comment by zsliu98** [comment]
> Hello guys, i hope you can read my post, i have tried this amazing plugin! I would like to know you will add the point frequency to each eq point! Right there is just a Note based point like, if you wanna eq bass you need to go to B1 to Eq, but i think freq number could help to achieve a more precise eq!

You can view the frequency if you set `Filter Parameter Tags Colour (Tag Colour)` as non-transparent (see https://zl-audio.github.io/plugins/zlequalizer/manual/#ui-setting-panel). If you want to set the frequency, you need to double-click the value on the knob (at the bottom panel) and enter the value.

> By the way which image format use the gui? o would like to propose a GUI for knobs and buttons!

The plugin doesn't use any images (except for the logo). Instead, it draws the knobs/buttons with ellipses and shadows. Therefore we can ensure display quality at any resolution.

**Reply by udeveloping** [reply]
oh nice!!! i will try this way!!!
